R/mod_marg.R

```diff
@@ -36,19 +36,32 @@ mod_marg2 <- function(mod, var_interest,
                       at = NULL, base_rn = 1,
                       at_var_interest = NULL){
 
+  stopifnot('glm' %in% class(mod))
```

Comment:
Moved this above the df recoding, because otherwise if mod isn't a glm object the df recoding can return weird errors.